packages/swap/src/routes/__tests__/quote.test.ts

```diff
@@ -386,19 +386,9 @@ describe('server', () => {
         ],
         poolInfo: [
           {
-            baseAsset: {
-              asset: 'ETH',
-              chain: 'Ethereum',
-            },
-            fee: {
-              amount: '200000',
-              asset: 'USDC',
-              chain: 'Ethereum',
-            },
-            quoteAsset: {
-              asset: 'USDC',
-              chain: 'Ethereum',
-            },
+            baseAsset: { asset: 'ETH', chain: 'Ethereum' },
+            fee: { amount: '0', asset: 'USDC', chain: 'Ethereum' },
```

Comment:
could deprecate the `fee` property on the type

Reply:
or even the whole `poolInfo` property

Reply:
was considering it, not sure if we want to continue showing range order fees but that requires an enhancement to the rpc, so i'll keep it as is for now without deprecating